Ignore a whole slew of sendmail options.  Probably some more left.

--- a/dma.c
+++ b/dma.c
@@ -1174,7 +1174,7 @@
 	snprintf(tag, 254, "dma");
 
 	opterr = 0;
-	while ((ch = getopt(argc, argv, "A:b:Df:iL:o:O:q:r:")) != -1) {
+	while ((ch = getopt(argc, argv, "A:b:B:C:d:Df:F:h:iL:N:no:O:q:r:R:UV:vX:")) != -1) {
 		switch (ch) {
 		case 'A':
 			/* -AX is being ignored, except for -A{c,m} */
@@ -1216,6 +1216,21 @@
 			doqueue = 1;
 			break;
 
+		/* Ignored options */
+		case 'B':
+		case 'C':
+		case 'd':
+		case 'F':
+		case 'h':
+		case 'N':
+		case 'n':
+		case 'R':
+		case 'U':
+		case 'V':
+		case 'v':
+		case 'X':
+			break;
+
 		default:
 			exit(1);
 		}
